recipes/bind/bind.inc

```diff
@@ -54,6 +55,6 @@ inherit binconfig
 BINCONFIG_FILES = "${bindir}/isc-config.sh"
 FILES_${PN}-dev += "${bindir}/isc-config.sh"
 DEPENDS_${PN}-dev = "libssl libcrypto ${DEPENDS_LIBC}"
-
-PACKAGES =+ "${PN}-utils "
+FILES_${PN}-config = "${sysconfdir}/bind/named.conf.local"
```

Comment:
Yes there are a number of configs, apart from the .conf.local. However, as I see it we dot he conf package in order to allow later override to provide machine/project specific configuration. And for that purpose only named.conf.local i relevant.
Please consider that including more files also force config packages in project/machine to include the same files.